bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/Connection.java

```diff
@@ -1111,14 +1108,7 @@ public void sendAnnouncement(Device device, String text, @Nullable String bodyTe
         } else {
             content.display.title = title;
         }
-        content.display.body = text;
-        if (text.startsWith("<speak>") && text.endsWith("</speak>")) {
-            content.speak.type = "ssml";
-            String plainText = text.replaceAll("<[^>]+>", "");
-            content.display.body = plainText;
-        } else {
-            content.display.body = text;
-        }
+        content.display.body = bodyText;
```

Comment:
Is this safe? `bodyText` is `@Nullable` while it was non-null before.

Reply:
You are right, it should not be marked as nullable. I changed it now.